scripts/ci/is-scanner-v2-available.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
I suggest to always use 
```suggestion
set -euo pipefail
```
unless there are strong reasons against some of the flags.
Here an article to learn about them: http://redsymbol.net/articles/unofficial-bash-strict-mode/

Reply:
Ok